### PR TITLE
Cjian/rc3

### PR DESCRIPTION
**Pull request by jchen351**

### CODE REVIEW

File: .pipelines/stages/jobs/steps/nuget-win-step.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ steps:
     DisplayName: 'ESRP - Sign C# dlls'
 
 - powershell: |
-    $VERSION = '0.1.0-rc1'
+    $VERSION = '0.1.0-rc3'
```
Comment (baijumeswani) [inline .pipelines/stages/jobs/steps/nuget-win-step.yml:19]: Can we read from VERSION_INFO file?

Comment (jchen351) [reply .pipelines/stages/jobs/steps/nuget-win-step.yml:19]: The rc format for python and nuget is different. Unless we create a different version info file.

**Review by baijumeswani — APPROVED** [review]

File: .pipelines/stages/jobs/steps/nuget-win-step.yml (hunk 1 above)
Comment (baijumeswani) [reply .pipelines/stages/jobs/steps/nuget-win-step.yml:19]: Yeah, is it possible to find a common ground between the two?

Comment (jchen351) [reply .pipelines/stages/jobs/steps/nuget-win-step.yml:19]: 0.1.0-rc3 vs 0.1.0rc3 we would need to have different suffix, but python packaging directly read from a file.

**Pull request merged**